data/src/main/java/com/moez/QKSMS/manager/WidgetManagerImpl.kt

```diff
@@ -33,7 +33,7 @@ class WidgetManagerImpl @Inject constructor(private val context: Context) : Widg
 
     override fun updateTheme() {
         val ids = AppWidgetManager.getInstance(context)
-                .getAppWidgetIds(ComponentName("dev.octoshrimpy.quik", "dev.octoshrimpy.quik.feature.widget.WidgetProvider"))
+                .getAppWidgetIds(ComponentName("com.unplugged.messages", "dev.octoshrimpy.quik.feature.widget.WidgetProvider"))
```

Comment:
@unpluggederan Is this the correct package name?

Reply:
Yes we talked about this before, we agreed it will be called "Messages" and the package name should reflect that.

Reply:
After discussion with Eran lets make it com.unplugged.sms